zone/bot_command.cpp

```diff
@@ -1793,14 +1856,32 @@ namespace MyBots
 				break;
 			}
 		}
-		if (!named_mob || !named_mob->GetGroup() || (!named_mob->IsClient() && !named_mob->IsBot()))
+		if (!named_mob || !MyBots::IsMyBot(bot_owner, named_mob) || (!named_mob->GetGroup() && !named_mob->GetRaid()) || (!named_mob->IsClient() && !named_mob->IsBot()))
 			return;
 
-		std::list<Bot*> group_list;
-		named_mob->GetGroup()->GetBotList(group_list);
-		for (auto member_iter : group_list) {
-			if (IsMyBot(bot_owner, member_iter))
-				sbl.push_back(member_iter);
+		if (bot_owner->IsRaidGrouped()) {
+			Raid* raid = bot_owner->GetRaid();
+			if (!raid)
+				return;
+			uint32 raid_group = raid->GetGroup(named_mob->CastToClient());
+			if (raid_group >= MAX_RAID_GROUPS) {
+				return;
+			}
+			for (int x = 0; x < MAX_RAID_MEMBERS; x++) {
```

Comment:
Can probably do
```cpp
for (const auto& m : raid->members) {
    if (m.member) {
```

Don't think we have to check if mob is in zone if we have a valid pointer.